docs/development/security/kampela_security_formalized.md

```diff
@@ -292,23 +292,27 @@ In case of companion app (or actually any wallet's) breach, confidentiality dama
 
```

Comment:
I think that attacks need to be considered additionally in the context of the owner - the device
1 Attacks or impacts hidden or unknown to the Owner
this is what follows in the text
2 Attacks or impacts using the Owner himself
These are any KNOWN TO THE OWNER AND REGISTERED IN THE SYSTEM facts of the relationship of unauthorized persons with the device (here, loss, direct transfer, etc.).
here i see 2 options
a) Enabling a direct disposal protocol (lost - returned, service life expired - no guarantee of maintaining functionality, etc.) These are most likely options for administrative conflict resolution and also our maintenance headache, but if the issue under item b is resolved - then all the same the protocol of remote utilization.
b) Enabling a remote disposal protocol (the technical possibilities of self-destruction of either the device or its contents should be considered, excluding the very possibility of using the contents of the device).